doc/element_model.tex

```diff
@@ -1,11 +1,9 @@
 A VOTable can provide serializations for an arbitrary number of data model
 types. In order to declare which models are represented in the file, data
 providers must declare them through the \texttt{MODEL} elements.
-Only models that are used in the file must be declared. A model is
-used if at least one element in the mapping block refers to it. In other terms, only models that define vodml-ids used in the
-annotation must be declared.
+All models that define vodml-ids used in the annotation must be declared.
 
-\begin{lstlisting}[caption={Example \texttt{MODEL} mapping block. (see in context line \ref{MODEL_snippet}) },language=XML]
+\begin{lstlisting}[caption={Example \texttt{MODEL} mapping block. (see line~\ref{MODEL_snippet} in Appendix A) },language=XML]
```

Comment:
Because I just notice it here: it's not pretty to hard-code "Appendix A" here.  It of course ought to be Appendix~\ref{app:example} or something like that.  I think that's too fragile to work at this position.  Perhaps sprinkling in a \protect helps?  Otherwise poke me and I'll invent something.

Reply:
done as suggested : Appendix~\ref{Appendix_A}